src/solvers/dgsem_p4est/dg_2d.jl

```diff
@@ -320,13 +412,21 @@ end
   # to obtain the flux of the large element.
   u_buffer .*= -2
 
-  # Copy interpolated flux values from buffer to large element face in the correct orientation
-  for i in eachnode(dg)
-    for v in eachvariable(equations)
-      # Use Tuple `node_indices` and `evaluate_index_surface` to copy flux
-      # to surface flux storage in the correct orientation
-      surface_index = evaluate_index_surface(large_indices, size_, 1, i)
-      surface_flux_values[v, surface_index, large_direction, large_element] = u_buffer[v, i]
+  # Copy interpolated flux values from buffer to large element face in the
+  # correct orientation.
+  # Note that the index of the small sides will always run forward but
+  # the index of the large side might need to run backwards for flipped sides.
+  if :i_backwards in large_indices
```

Comment:
Again, why is it not possible to use the same `start,step` logic here as elsewhere?

Reply:
See above